libraries/botbuilder/src/skills/cloudSkillHandler.ts

```diff
@@ -71,9 +71,9 @@ export class CloudSkillHandler extends CloudChannelServiceHandler {
      * This method allows you to send an activity to the end of a conversation.
      *
      * This is slightly different from replyToActivity().
-     * * sendToConversation(conversationId) - will append the activity to the end
+     * sendToConversation(conversationId) - will append the activity to the end
      * of the conversation according to the timestamp or semantics of the channel.
-     * * replyToActivity(conversationId,ActivityId) - adds the activity as a reply
+     * replyToActivity(conversationId,ActivityId) - adds the activity as a reply
```

Comment:
I think we should use the `-` here to keep the sub-bullet format.

Reply:
fixed